app/api_alpha/views.py

```diff
@@ -959,9 +960,6 @@ class DiffView(APIView):
         }
     )
     def get(self, request):
-        # the day the quantity of data will be too big, we could stream the response
-        # see https://docs.djangoproject.com/en/5.0/howto/outputting-csv/#streaming-csv-files
-
         since_input = request.GET.get("since", "")
```

Comment:
```suggestion

    # Alternative idea:
    # In case the streaming solution becomes a problem (might be because of the process fork or any other reason), here is another idea for avoiding relying too heavily on the database:
    # Since the list of modifications between two dates is static, we could precalculate large time chunks (eg: one each month) and save those results in CSV files. When the diff endpoint is requested, we could assemble some of those large CSV chunks into one, add remaining rows to complete the time period by fetching them from db and finally serve the combined file. 

        since_input = request.GET.get("since", "")
```

Reply:
Remarque sur la remarque : dans les discussions préliminaires à propos de ce endpoint, nos interlocuteurs avaient parlé de rajouter des filtres (par bounding box, par code insee, par département, etc),  ce qui limiterait la possibilité de pré-calculer quoi que ce soit.